libiocage/lib/Host.py

```diff
@@ -91,4 +91,4 @@ def processor(self):
 
 class Host(HostGenerator):
 
-    class_distribution = libiocage.lib.Distribution.DistributionGenerator
+    _class_distribution = libiocage.lib.Distribution.DistributionGenerator
```

Comment:
is this supposed to be a generator?
is it supposed to be `class_distribution` and not `_class_distribution`

Reply:
As rule of thumb: when the class is not a generator it is not supposed to expose or use generators at all. So, this should be a Distribution, not DistributionGenerator.